### ISSUE TITLE
How machine learning can be combined with Python coding

### ISSUE THREAD

**Issue by iampossessed** [iampossessed/globalaltruism#3]
I may be asking a silly question, but please understand. I am just a beginner who wants to know as much as possible about that. So my question is in my title. Thankful to any opinions.

**Comment by iampossessed** [comment]
For example, I heard that ML is an area of wider (not wilder!) field of artificial intelligence that makes use of statistical models to make forecasts. Who can explain this in plain English?

**Comment by SabinaWellington** [comment]
I am not that much experienced in it either. So I won't be able to bind it with Python coding. Better if you ask professionals. But as for machine learning, I can give you an example, based on [crowd surge analysis](https://www.padme.ai/post/crowd-surge-analysis) that I have recently read about. You can make a crowd surge analysis and use this information to predict  better sales. You won't have to analyze those complex tabular reports. That will be done by AI. So I am sure you will have more questions. So as I said, ask professionals.
